web/modules/custom/usergreeting/src/GreetingTime.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\usergreeting;
+
+use Drupal\Component\Datetime\TimeInterface;
+
+/**
+ * Provides a greeting service for users dependent on the time of  day.
+ */
+class GreetingTime {
```

Comment:
Is there any class that extends this class? If not, it's a good practice to make it `final`.